modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -19,16 +19,16 @@ To configure and use the tokenization service, you must:
  ** Tokenization *Manage Tokenization Services* 
 + 
 See xref:2.x@api-manager::environment-permission-task.adoc[Granting Permission to Access an Environment]
-. xref:runtime-fabric::install-create-rtf-arm.adoc[Install Runtime Fabric]. Anypoint Runtime Fabric is a container service that automates the deployment and orchestration of Mule applications and API gateways.
-. To allow Mule applications or API gateways to listen on inbound connections, enable inbound traffic, you must xref:runtime-fabric::enable-inbound-traffic.adoc[Enable inbound traffic] on Runtime Fabric.
+. xref:runtime-fabric::install-create-rtf-arm.adoc[Install Runtime Fabric]. Anypoint Runtime Fabric is a container service that automates the deployment and orchestration of Mule apps and API gateways.
+. To allow Mule apps or API gateways to listen on inbound connections, you must xref:runtime-fabric::enable-inbound-traffic.adoc[enable inbound traffic] on Runtime Fabric.
```

Comment:
The previous bullet joins apps and gateways using "and" and this bullet uses "or"; is that an intentional distinction?